labs/06/3d_recognition.py

```diff
@@ -3,12 +3,13 @@
 import tensorflow as tf
 
 class Dataset:
-    def __init__(self, filename):
+    def __init__(self, filename, shuffle_batches = True):
         data = np.load(filename)
         self._voxels = data["voxels"]
         self._labels = data["labels"] if "labels" in data else None
 
-        self._permutation = np.random.permutation(len(self._voxels))
+        self._shuffle = shuffle_batches
```

Comment:
Could you please rename it to _shuffle_batches, mostly for consistency with the argument name?